feat :Implemented code changes for login and forgot password page

richardgidado opened the pull request:
feat :Implemented code changes for login and forgot password page

richardgidado commented:
Update on layout for small screens

malimikillua commented on src/app/login/forgotpassword/page.tsx:
```diff
@@ -0,0 +1,107 @@
+"use client"
+
+import { ArrowLeftIcon } from "@heroicons/react/24/outline";
```
This package is not in the package.json causing build to fail, please add and push

malimikillua commented on src/app/login/forgotpassword/page.tsx:
```diff
@@ -0,0 +1,107 @@
+                {/* Left Section: Branding */}
+                <div className="w-1/2 text-black p-2 flex flex-col justify-between h-85">
+                    <div className="w-95 h-80 flex items-start">
+                        <svg width="337" height="290" viewBox="0 0 337 439" fill="none" xmlns="http://www.w3.org/2000/svg">
```
download svg as an image and import it as such

malimikillua commented on src/app/login/page.tsx:
```diff
@@ -1,9 +1,123 @@
+        <div className="w-1/2 text-black p-2 flex flex-col justify-between h-85">
+          <div className="w-95 h-80 flex items-start">
+            <svg width="337" height="290" viewBox="0 0 337 439" fill="none" xmlns="http://www.w3.org/2000/svg">
+              <path d="M5.75702 157.133C9.0288 153.479 12.1766 149.712 15.6068 146.195C26.7056 134.848 37.8595 123.552 49.0685 112.306C71.0342 90.1725 93.1033 68.1381 114.938 45.8926C120.641 40.074 127.171 36.6105 135.464 35.8188C162.975 33.18 190.106 33.9254 216.246 43.8475C217.892 44.4421 219.502 45.1249 221.068 45.8926C224.966 47.8718 228.583 47.443 232.24 45.1999C252.316 32.8853 272.418 20.59 ...
```
actually since left section brading is the same for both screens, make this into a component, and make sure you don't use inline svg

richardgidado commented:
Created resusable components for brandind and form,installed heroicons package,removed responsive warning messages

richardgidado commented:
Added a reusable component for branding and forms for sign in and forgot password page,removed responsive warning messages,installed heroicons package,used an image instead of svg in the branding component
